### ISSUE TITLE
Test failure related to metamath-program.zip

### ISSUE THREAD

**Issue by jkingdon** [metamath/set.mm#2979]
The test run https://github.com/jkingdon/set.mm/actions/runs/3862758965/jobs/6584526336 is failing at the step

```
configure.ac:33: error: required file 'src/Makefile.in' not found
```

It is not clear what changed (if anything) to make this start failing. At https://github.com/metamath/set.mm/pull/2977#issuecomment-1374539489 we have a report of this symptom which went away on rebuild. Which maybe means it has something to do with some kind of caching or which build machine the job ran on or (I'm a bit grasping at straws, this particular error message doesn't at first glance appear to be the kind of thing which would be intermittent).

Of course if the https://github.com/metamath/metamath-exe/zipball/master endpoint (used in `scripts/download-metamath`) returns different results on different runs, that would explain it.

**Comment by benjub** [comment]
Thanks @jkingdon for opening this issue. With the more recent pull requests, it appears that the failure is systematic, and then the test is systematically re-run for some reason and this works.

**Comment by david-a-wheeler** [comment]
My fault. I changed how metamath is downloaded (it's downloaded directly from GitHub), and that results in some subtle changes. I think this fixes things:
https://github.com/metamath/set.mm/pull/2982

**Comment by jkingdon** [comment]
> My fault. I changed how metamath is downloaded (it's downloaded directly from GitHub), and that results in some subtle changes. I think this fixes things: #2982

Can't say I understand what is going on here (in particular, why the build would sometimes pass and sometimes not), but I guess if it works with those changes, I won't complain.

**Comment by jkingdon** [comment]
I haven't seen the `'src/Makefile.in' not found` symptom in a few days but I am seeing a different one which I opened as a new issue: https://github.com/metamath/set.mm/issues/2986
